src/libraries/Microsoft.Extensions.FileProviders.Physical/tests/Microsoft.Extensions.FileProviders.Physical.Tests.csproj

```diff
@@ -2,9 +2,35 @@
 
   <PropertyGroup>
     <RootNamespace>Microsoft.Extensions.FileProviders.Physical</RootNamespace>
-    <TargetFrameworks>$(DefaultNetCoreTargetFramework);net472</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent);$(NetFrameworkCurrent)</TargetFrameworks>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Include="**\*.cs" />
+    <Compile Include="$(CommonPath)Extensions\Microsoft.AspNetCore.Testing\src\xunit\OSSkipConditionAttribute.cs">
```

Comment:
I believe you might be missing TestingUtils in this path.  Can you fix that up and build these projects to make sure they're fixed before pushing?